### ISSUE TITLE
Styles imported through angular.json blocked by Content-Security-Policy script-src: self

### ISSUE THREAD

**Issue by karun19** [angular/angular#49096]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

No

### Description

I have imported couple of css and scss files through angular.json styles array, this eventually added under . I have **content-security-policy: object-src 'none'; script-src 'self'**, which is blocking the above from loading and hence application not working. The error is as below,

**Refused to execute inline event handler because it violates the following Content Security Policy directive: "script-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.** 

I see adding 'unsafe-inline' to CSP will solve the problem but I cannot go with this solution as this is unsafe.

Plesae suggest me any other solution which is not a threat to security.

Also pleae throw some light on why css is being blocked by script-src policy.

### Please provide the exception or error you saw

```true
Refused to execute inline event handler because it violates the following Content Security Policy directive: "script-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-...'), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular: 13.2.3
node: v16.13.1
```

**Comment by alan-agius4** [comment]
Duplicate of https://github.com/angular/angular-cli/issues/20864
